t5/convert.py

```diff
@@ -1,75 +0,0 @@
-import numpy as np
```

Comment:
I think the 23 downloads are from your nice [open_sora](https://github.com/awni/mlx-examples/tree/open_sora/open_sora) example. That should still work OK since it has it's own T5 loading so we're probably good for the moment.